### DISCUSSION TITLE
Building on Windows

### DISCUSSION THREAD

**Discussion by bradlarsen** [VectorCamp/vectorscan#314] [Building] [answered]
Is it feasible to build on Windows?

I see that in the November 2023 release, pkgconfig was made a requirement (#188). How does one get pkgconfig on Windows?

Thanks!

**Comment by markos** [comment, marked as answer]
Hi @bradlarsen,
If you can build on Windows using WSL, that might work, but again it's an unsupported platform. We do not expect to change this in the near future.

**Reply by bradlarsen** [reply]
@markos thanks for clarifying!

Windows support is not a high priority for me, but it would be nice to have, as then I could put out [Nosey Parker](https://github.com/praetorian-inc/noseyparker) releases for Windows.

If I could get Vectorscan building on Windows, would you be interested in taking a PR for that? The pkg-config requirement is a problem at present, but I'm not sure what else there might be. (The original Hyperscan codebase _does_ appear to build on Windows.)
